### PR TITLE
[NOD-517] Update doc.go files

### PR DESCRIPTION
**Pull request by stasatdaglabs**

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -1,121 +1,20 @@
-// Copyright (c) 2013-2017 The btcsuite developers
```
Comment (svarogg) [inline doc.go]: The main doc.go actually does need the copyright notice.

File: doc.go [hunk 2]
```diff
@@ -1,121 +1,26 @@
...
-                            default settings for the active network.
-      --rejectnonstd        Reject non-standard transactions regardless of the
-                            default settings for the active network.
-
-Help Options:
-  -h, --help           Show this help message
+For an up-to-date help message:
+  kaspad --help
 
+An interesting point to note is that the long form of all option flags
```
Comment (someone235) [inline doc.go]: Delete "An interesting point to note is that "

File: logs/LICENSE [hunk 3]
```diff
@@ -1,15 +0,0 @@
-ISC License
-
-Copyright (c) 2013-2014 Conformal Systems LLC.
```
Comment (someone235) [inline logs/LICENSE]: Please add it to the main copyright section

File: rpcclient/doc.go [hunk 4]
```diff
@@ -123,7 +119,7 @@ The third category of errors, that is errors returned by the server, can be
 detected by type asserting the error in a *rpcmodel.RPCError. For example, to
 detect if a command is unimplemented by the remote RPC server:
 
-  amount, err := client.GetBalance("")
+  amount, err := client.GetNetTotals()
```
Comment (someone235) [inline rpcclient/doc.go]: GetNetTotals doesn't return amount

**Pull request merged**